web3/secureTransport.js

```diff
@@ -74,6 +74,7 @@ export function createSecureTransport () {
   });
 
   return {
+    ready: Promise.resolve(null),
```

Comment:
Where is this used?

Reply:
Somewhere inside the new Parity codebase.

Reply:
Ah ok. But so it's OK if it's always `null`?

Reply:
Yes. Couldn't find it now in current Parity codebase, I need to check if it's still necessary (might be needed for 1.7/1.8 compatibility).

Reply:
Okay, that's fine :)